### PR TITLE
Update "new API" instructions

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

File: NEWAPI.md [hunk 1]
```diff
@@ -32,124 +31,222 @@ know how to use `git` appropriately.
...
 
-Run `./build.sh Your.ApiName.Here` to check it builds and the
-generated unit tests pass.
+```sh
+./build.sh Google.Cloud.Dialogflow.Cx.V3
+```
+
+If anything fails here, it's likely to be either an API
+configuration problem, or a generator problem. Either way, raise a
+GitHub issue - unless you're feeling particularly patient, it's
```
Comment (amanda-tarafa) [inline NEWAPI.md:176]: nit: inpatient?

File: NEWAPI.md [hunk 2]
```diff
@@ -32,124 +31,222 @@ know how to use `git` appropriately.
...
+**Please review smoke tests before running or committing them.**
+In particular, if the RPC sounds like it might be "dangerous" based
+on the name (such as the Vision API's `PurgeProducts` method),
+you'll want to look really, really carefully.
 
-Step 10 (Optional): Release the first package for the API
+Once you've reviewed the smoke tests, enable the API in the
+[Cloud Console API
+dashboard](https://console.cloud.google.com/apis/dashboard) and then
+run them with release manager. To run the smoke tests, use the
```
Comment (amanda-tarafa) [inline NEWAPI.md:220]: nit: Maybe add a note here about default application credentials?

**Review by amanda-tarafa — APPROVED** [review]
LGTM

File: NEWAPI.md (hunk 1 above)
Comment (jskeet) [reply NEWAPI.md:176]: Nope, this is deliberate. *Unless* you're feeling particularly patient (as in, you're willing to do a lot of debugging work yourself)

File: NEWAPI.md (hunk 2 above)
Comment (jskeet) [reply NEWAPI.md:220]: Yup, have added just a line saying it assumes they're set up.

File: NEWAPI.md (hunk 1 above)
Comment (amanda-tarafa) [reply NEWAPI.md:176]: Ha, I thought it was meant to say "Unless you're feeling particularly impatient" (as in, you don't want to wait for someone in the team to look at it), haha.

**Pull request merged**